apps/ocp-plugin/deploy/helm/flightctl-ocp-ui/values.yaml

```diff
@@ -1,12 +1,17 @@
-flightctlUi:
-  namespace: flightctl
-  image: quay.io/flightctl/flightctl-ocp-ui:latest
-  flightctlServer: https://flightctl-api:3443
-  flightctlMetricsServer: https://flightctl-api:9090
-  flightctlGrpcServer: flightctl-api-agent-grpc:7444
-  bootcImgUrl: "" #Bootc device image. Example: quay.io/example/example-agent-centos:bootstrap
```

Comment:
We'd need to make changes to the "Add devices" modal if we're not supporting any option for adding devices, and checkout the UX with Nir.

And ideally also remove the references to the two properties in the rest of the codebase (proxy, AppContext, etc)